diagnostic_aggregator/CMakeLists.txt

```diff
@@ -3,9 +3,8 @@ cmake_minimum_required(VERSION 2.8.3)
 project(diagnostic_aggregator)
 
 # Load catkin and all dependencies required for this package
-find_package(catkin REQUIRED diagnostic_msgs pluginlib roscpp rospy rostest xmlrpcpp)
-
-catkin_package(DEPENDS diagnostic_msgs pluginlib roscpp rospy xmlrpcpp
+find_package(catkin REQUIRED diagnostic_msgs pluginlib roscpp rospy rostest xmlrpcpp bondcpp bondpy)
+catkin_package(DEPENDS diagnostic_msgs pluginlib roscpp rospy xmlrpcpp bondcpp bondpy
```

Comment:
bondpy isn't using during the C++ compilation process or by downstream C++ compiles, so this doesn't need to find_package bondpy.